db/migrate/20170302234140_make_task_id_a_uuid.rb

```diff
@@ -0,0 +1,11 @@
+class MakeTaskIdAUuid < ActiveRecord::Migration
+  def change
+    add_column :tasks, :uuid, :uuid, default: "uuid_generate_v4()", null: false
+
+    change_table :tasks do |t|
+      t.remove :id
+      t.rename :uuid, :id
+    end
+    execute "ALTER TABLE tasks ADD PRIMARY KEY (id);"
+  end
```

Comment:
Since this is all still one PR/body-of-work it might have been reasonable to just correct the original migration.